### ISSUE TITLE
No way to turn arrows (engine best move) off?

### ISSUE THREAD

**Issue by TapioOfTapiola** [lukasmonk/lucaschessR2#150]
See subject. In simply a match, the engine thinks shows its best move as an arrow - I found no way to turn it off. Is there really no way? Thx!

**Comment by TapioOfTapiola** [comment]
If it's the option "Arrow with the best move..." then it's broken, upon each new evaluation, an arrow appears again.
Also, those arrows are so hugeee! I wish there was a more subtle approach. E.g. last move arrow. If it e.g. would just put a border around source and target field...

**Comment by lukasmonk** [comment]
Options/colors/Main board.

In analysis, Variations tab, to configure.

**Comment by TapioOfTapiola** [comment]
> Options/colors/Main board.

Yes, that would be the "Active move" arrow. But even if I give it 99% transparency (which is a kind of awkward way just to hide the thinking arrows) - the arrow is shown while Stockfish is thinking.

Have you tried it? I'm in a match vs. engine setup here.

**Comment by lukasmonk** [comment]
![imagen](https://github.com/lukasmonk/lucaschessR2/assets/6254278/4779dbfb-e4ef-4133-ab3a-41fc8f4f82f9)

**Comment by TapioOfTapiola** [comment]
Overlooked it, maybe I didn't expect your software to be so well layed-yout. 💯  Really well done and many thanks.
